.github/workflows/release.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         toolchain: stable
         target: ${{ matrix.target }}
     - name: Build release binary
-      run: cargo build --target ${{ matrix.target }} --verbose --release --features vendored-openssl
+      run: cargo build --target ${{ matrix.target }} --verbose --release --features vendored-openssl,packaging
```

Comment:
You could alphabetize the features listed here, at your preference.

Reply:
Makes sense.